src/main/java/org/webonise/concurrent_modification_exception/concurrent_modification_with_resolving/Application.java

```diff
@@ -0,0 +1,31 @@
+package org.webonise.concurrent_modification_exception.concurrent_modification_with_resolving;
```

Comment:
Wrong convention of package name. Please go through the standards followed to define package name